bundles/mapping/mapwfs2/request/ShowOwnStyleRequestHandler.js

```diff
@@ -15,6 +15,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.mapwfs2.request.ShowOwnStyleRequ
         this.plugin = plugin;
         this.localization = plugin.getLocalization('popup');
         this.visualizationForm = this.plugin.getVisualizationForm();
+        this._usesMvtPlugin = this.plugin._clazz === 'Oskari.wfsmvt.WfsMvtLayerPlugin';
```

Comment:
We could move the implementtation specific behaviour to the implementing plugins to avoid this.